cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/SqlTransactionRunner.java

```diff
@@ -59,7 +59,7 @@ public void run(TxRunnable runnable) throws TransactionException {
     try {
       connection = dataSource.getConnection();
     } catch (SQLException e) {
-      throw new TransactionException("Unable to get connection to the sql database", e);
+      throw new SqlTransactionException(e, "Unable to get connection to the sql database.");
```

Comment:
nit: Follow the regular exception constructor code style: message first, exception second

Reply:
Ack